thyrel-web/src/components/room/lobby/PlayerCard.tsx

```diff
@@ -64,7 +64,7 @@ export default function PlayerCard({
         ) : (
           isKickable && (
             <button
-              onClick={() => onKick?.(id)}
+              onClick={() => window.confirm(`Do you really want to kick ${name} ?`)&&onKick?.(id)}
```

Comment:
si ton test passe pas c'est à cause de ça,
il ne faudrait pas que ce soit ici qu'il y ai un confirm (même si c'est juste mais moins facile à tester)
tu peux passé en params du `onKick` l'id + le nom (en changeans le type dans les props évidement) et mettre le window.confirm dans 
`<PlayerCard onKick={(id, name) => window.confirm("kick the name?") && onKick(id)}`

Reply:
enfaite tu déplace la logic chez le parent à la place d'ici